7/main.js

```diff
@@ -0,0 +1,168 @@
+const carmileage = {};
+function newMemberRegister(
+  teamname,
+  drivername,
+  birthdate,
+  carbrand,
+  sponsorname,
+  useraccess
+) {
+  if (
+    teamname == undefined ||
+    drivername == undefined ||
+    birthdate == undefined ||
+    carbrand == undefined ||
+    sponsorname == undefined ||
+    useraccess == undefined
+  ) {
+    return "Введіть всі дані.";
+  } else {
+    if (
+      typeof teamname !== "string" ||
+      typeof drivername !== "string" ||
+      typeof carbrand !== "string" ||
+      typeof sponsorname !== "string" ||
+      typeof useraccess !== "boolean" ||
+      typeof birthdate !== "number"
+    ) {
+      return "Введіть дані коректно.";
+    } else {
+      if (!useraccess) {
+        return "Вас не допущено до пробігу.";
+      } else {
+        return {
+          useraccess: useraccess,
+          teamname: teamname,
+          drivername: drivername,
+          driverbirthdate: birthdate,
+          carbrand: carbrand,
+          sponsorname: sponsorname,
+        };
+      }
+    }
+  }
+}
+
+carmileage.some_user = newMemberRegister(
+  "Dota2",
+  "Danylo",
+  1969,
+  "V0lve",
+  "Volodya",
+  true
+);
+
+const informationabouttheusers = {};
+function newUserCreator() {
+  const login = prompt("Введіть свій логін.");
+  const password = prompt("Введіть свій пароль.");
+  const city = prompt("Введіть своє місто.");
+  const country = prompt("Введіть назву країни у якій ви проживаєте.");
+  const sex = prompt("Введіть вашу стать.");
+  const age = prompt("Введіть ваш вік.");
+  if (
+    login === null ||
+    password === null ||
+    city === null ||
+    country === null ||
+    sex === null ||
+    age === null
+  ) {
+    return "Введіть всі дані.";
```

Comment:
In that case, we must re-run all prompts. `while` or `do...while` would do the work